.github/workflows/on-pull-request.yaml

```diff
@@ -38,3 +38,52 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           fail_ci_if_error: true
+  e2e-tests:
+    name: Run e2e tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Download kind
+        run: |
+          curl --silent -LO https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64
+          curl --silent -LO https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64.sha256sum
+          cat kind-linux-amd64.sha256sum | sha256sum --check
+      - name: Install kind
+        run: |    
+          chmod +x ./kind-linux-amd64
+          sudo mv ./kind-linux-amd64 /usr/local/bin/kind
```

Comment:
Do these need to be separate steps?

Also, what about just using one of these?
  - https://github.com/marketplace/actions/kind-kubernetes-in-docker-action
  - https://github.com/marketplace/actions/kind-cluster

Reply:
I like the approach of wrapping it all in an `npm` action https://github.com/kubev2v/forklift-console-plugin/pull/350#discussion_r1148501711 , so the workflow looks like:
```
npm run test
npm run lint
npm run e2e
```

Reply:
@sjd78 
> Do these need to be separate steps?
>
This seemed the easiest way to fail the job if the checksum is incorrect. 

> Also, what about just using one of these?
>
>    https://github.com/marketplace/actions/kind-kubernetes-in-docker-action
>   https://github.com/marketplace/actions/kind-cluster

First, I didn't know that they exist:) 
Second, using the same initialization in local dev env and in GH actions has some value(i.e. I've tested the flow on my local VM in standard bash ).